### PR TITLE
Space Travelers.

### PR DESCRIPTION
**Pull request by wadaudzai**
Dear Reviewer.

This project was built to display three main components: Rockets, Missions, and Profiles. The first component of this website takes the data from SpaceX API about the rockets it also features you with a reservation button in which you can reserve the rockets. The second component of this project shows a missions page which also takes the data from SpaceX API in order to show the missions and give details about them. Users are given the opportunity to join these missions and be part of space history. The third part of this project is built to display a profile where the users can see which rockets they have reserved and which mission they have joined.

Dear reviewer, we have worked on this project to the best of our knowledge please check it if you got time. Thanks for your time.

Regards,

### CODE REVIEW

File: src/components/Missions/Mission.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                  <td>{mission.mission_name}</td>
+                  <td>{mission.description}</td>
+                  <td>
+                    <Membership reserved={!!mission.reserved} />
+                  </td>
+                  <td>
+                    <Join
+                      reserved={!!mission.reserved}
+                      id={mission.mission_id}
+                    />
```
Comment (uwadonat) [inline src/components/Missions/Mission.js:40]: - [ ] When I navigate from "Mission" tab to "Rocket" tab the reserved mission is canceled. Please fix this to make your project more perfect.

File: src/components/tests/Mission.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  test('Missions snapshot.', () => {
+    const missionTest = render(
+      <Provider store={store}>
+        <Router>
+          <Missions />
+        </Router>
+      </Provider>,
+    );
+    expect(missionTest).toMatchSnapshot();
+  });
```
Comment (uwadonat) [inline src/components/tests/Mission.test.js:18]: - [ ] Some of your tests are failing. Please fix this.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Professional readme file ✔️
- Well organized and clean project ✔️
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/tests/Mission.test.js (hunk 2 above)
Comment (wadaudzai) [reply src/components/tests/Mission.test.js:18]: Fixed.

File: src/components/Missions/Mission.js (hunk 1 above)
Comment (wadaudzai) [reply src/components/Missions/Mission.js:34]: Fixed by Dear @Ukaypromise

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @wadaudzai and @Ukaypromise 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f313149537762674378457a4d79592f67697068792e676966


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**